doc/pages/keys.asciidoc

```diff
@@ -570,10 +570,10 @@ to skim through the jump list using:
     select first and last characters of each selection
 
 *C*::
-    copy the main selection to the next line
+    duplicates selections on the lines that follow them
```

Comment:
This doc uses imperative mood (?, I am not a native speaker) so it should be "duplicate selections" etc. Same for the docstrings in `normal.cc`